layouts/html.hbs

```diff
@@ -132,7 +136,13 @@
           }));
           iframeLoadedResolve();
         },
-        onMessage: window.isOverlay ? window.Overlay.onMessage : function() {},
+        onMessage: (message) => {
+          if (message.token) {
+            iframeJWTResolve(message.token);
```

Comment:
does an error occur if iframeJWTResolve is resolved multiple times?

Reply:
Nope. Also, only the first resolve sets the promise's fulfilled value